### PR TITLE
lmdb: Fix stack-allocated string and update python-lmdb

### PR DESCRIPTION
**Pull request by thomasfire**

### CODE REVIEW

File: mingw-w64-lmdb/002-fix-stack-allocated-string.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
++++ b/libraries/liblmdb/mdb.c
+@@ -1735,13 +1735,10 @@
+ mdb_strerror(int err)
+ {
+ #ifdef _WIN32
+-	/** HACK: pad 4KB on stack over the buf. Return system msgs in buf.
+-	 *	This works as long as no function between the call to mdb_strerror
+-	 *	and the actual use of the message uses more than 4K of stack.
+-	 */
++    // Fix the return of stack-allocated string
```
[the remark this answers is not in the document]
Comment (thomasfire) [reply mingw-w64-lmdb/002-fix-stack-allocated-string.patch:12]: Yes, but before it was stack allocated, so I fixed it

**Comment by thomasfire** [comment]
Notified LMDB devs: https://bugs.openldap.org/show_bug.cgi?id=10050

**Comment by Biswa96** [comment]
Please bump the pkgrel values in both PKGBUILD files.

File: mingw-w64-lmdb/002-fix-stack-allocated-string.patch (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmuetzel) [reply mingw-w64-lmdb/002-fix-stack-allocated-string.patch:12]: Your fix looks fine. 
But I agree that the comment might be confusing when upstreamed if it is referring to code that was there but is no longer "visible".
Maybe, something along the lines of "Avoid potential issue with stack-allocated string by using static" would be clearer?

**Pull request merged**